### PR TITLE
Add link to types docs

### PR DESCRIPTION
**Pull request by Stefie**
Addresses #1017 

I have a PR open in the API repo that is adding a lot of additional Info to the linked page (short description for all types) https://github.com/polkadot-js/api/pull/849

### CODE REVIEW

**Comment by Stefie** [comment]
@xlc Is that what you meant by 

>  type definition syntax page 

or were you thinking of something else in #1017?

**Comment by xlc** [comment]
I mean a document describe what should be put in the JSON editor.
And describe special syntax like `_enum`. Later there will be more special syntax like one for array `[u8; 16]`

**Comment by Stefie** [comment]
Ah, no, there's no real documentation for that, only this short paragraph in the api-docs which gives some idea what format people need to use. https://polkadot.js.org/api/api/#registering-custom-types

And the actual `createType.ts` and `Enum.ts` files in the codec, which you contributed to so I'm assuming that you're more familiar with it than me.

**Comment by Stefie** [comment]
Since we don't have proper docs yet, I added an info overlay and a link to the types api docs for now. 

I also opened a ticket in apps to add documentation for typeRegistry. https://github.com/polkadot-js/api/issues/856 to address:
> I mean a document describe what should be put in the JSON editor.
And describe special syntax like _enum. Later there will be more special syntax like one for array [u8; 16]

File: packages/app-settings/src/Developer.tsx [hunk 1]
```diff
@@ -64,16 +65,24 @@ class Developer extends React.PureComponent<Props, State> {
...
                 className='editor'
                 code={code}
                 isValid={isJsonValid}
                 onEdit={this.onEditTypes}
               />
-            </Labelled>>
+            </Labelled>
           </div>
         </div>
+        <div className='ui--row info'>
```
Comment (jacogr) [inline packages/app-settings/src/Developer.tsx]: This should really be part of a `HelpOverlay`, i.e. like done in https://github.com/polkadot-js/apps/pull/1030/files#diff-1149998f281026a5e26b8bc8a0e0e64fR1

(For now would just remove this info row - but should be added to fully close the original logged issue, but shouldn't stand in the way of just the basic help added above)

Comment (Stefie) [reply packages/app-settings/src/Developer.tsx]: Agreed, but we just don't really have the help content yet so I thought a link to the API docs would be better than nothing.  But I'm also fine with removing it.

**Comment by Stefie** [comment]
Somehow the Travis CI Pipeline doesn't get triggered on merge commits. Is there a way to start it manually?

**Comment by jacogr** [comment]
Yes, https://travis-ci.com/polkadot-js/apps - you should have access since you have access to this repo.

**Comment by Stefie** [comment]
Yeah, I know Travis and how to re-run a task, but how do I initialize a build on a branch? 

The last Job from my branch was triggered from commit `988dddc`  https://travis-ci.com/polkadot-js/apps/branches .
The merge commit just didn't trigger a job at all, it just hung up on `Expected — Waiting for status to be reported`.  That happens to me regularly, my workaround is always to just push an insubstantial change after the merge commit.

> https://travis-ci.com/polkadot-js/apps/requests?requestId=199461088

I'm getting a 500 internal server error on that link

**Comment by jacogr** [comment]
Well, not even sure how to find that link again now. However what it said (paraphrasing) "the build was cancelled due to skip directive" (or something like that). So it found "[CI skip]" in the commit and didn't run it.

**Comment by Stefie** [comment]
Hmm, strange, ok, well, it's building now... Does that happen to you or is it just me?
I'll keep an eye on it.

**Comment by jacogr** [comment]
Has hapenned a couple of times (I think), but never enough to drive me crazy.

**Comment by Stefie** [comment]
Not crazy, just mildly annoyed since a merge PR it's often my last PR, but since the workaround is working it's fine I guess...

Any other changes or can we merge this?

**Pull request merged**